components/board/genericlinux/board.go

```diff
@@ -492,6 +492,18 @@ func (b *Board) WriteAnalog(ctx context.Context, pin string, value int32, extra
 	return nil
 }
 
+// StreamTicks starts a stream of digital interrupt ticks.
+func (b *Board) StreamTicks(ctx context.Context, interrupts []string, ch chan board.Tick, extra map[string]interface{}) error {
+	for _, name := range interrupts {
+		interrupt, ok := b.DigitalInterruptByName(name)
+		if !ok {
+			return errors.Errorf("unknown digital interrupt: %s", name)
```

Comment:
If the second interrupt returns with errors, the first will still have the callback added. Is that a problem? If so, split this into a loop that constructs a list of interrupts from the list of names (which might return errors if an interrupt cannot be found), and a second loop that adds the callback to everything in the list.